packages/client/src/client.ts

```diff
@@ -1765,18 +1833,28 @@ export class SkipClient {
       }
       return calculateFee(Math.ceil(parseFloat(estimatedGasAmount)), gasPrice);
     });
-    const feeBalance = await this.balances({
-      chains: {
-        [chainID]: {
-          address: signerAddress,
-          denoms: feeAssets.map((asset) => asset.denom),
-        },
-      },
+
+    const feeAssetFoundInSkipBalances = feeAssets.some((asset) => {
```

Comment:
should we use `.every` here

Reply:
because if it's undefined we should fetch the balance

Reply:
i was originally thinking they dont need all of the feeAssets, if they have one i believe they can pay with whatever asset they have but it complicates things if they have it but it's not enough... and I think in practice we most likely will have all so I think we can update to using every